.terraform.lock.hcl

```diff
@@ -2,21 +2,22 @@
 # Manual edits may be lost in future updates.
 
 provider "registry.terraform.io/equinix/metal" {
-  version = "1.0.0"
+  version = "3.1.0"
```

Comment:
@displague Why is it important to keep this in the repo? It changes a lot. And gets overwritten when you do a terraform init it seems like it doesn't help us... Or am I wrong?

Reply:
The current best practices for a Terraform module are to leave the `provider {}` definitions out of the module. Require the user to define that.  Your module still has requirements on what provider versions are acceptable and `.terraform.lock.hcl` defines that.

We could keep the version out of the file, or we can pin it to something more flexible, like `= "> 3.0.0, < 4.0.0"` (whatever the lower bound actually is).